Doppler.MercadoPagoApi/Controllers/PaymentController.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Doppler.MercadoPagoApi.Controllers
+{
+    [Authorize]
```

Comment:
Take care of this.

Take into account that this annotation only defines that the controller requires a valid Doppler token.

If you want to restrict it to _Doppler applications_, you can use `[Authorize(Policies.ONLY_SUPERUSER)]`. If you want to allow access only to the ower of the resource (or _Doppler applications_), you can use `[Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]` and the URL should start with `/accounts/{accountname}/`.

You can find examples in https://github.com/FromDoppler/doppler-mercadopago-api/blob/main/Doppler.MercadoPagoApi/Controllers/HelloController.cs and test them https://github.com/FromDoppler/doppler-mercadopago-api/blob/main/demo.http

Reply:
Yes @andresmoschini , my idea is include the policies when we will add the endpoints. The idea of this task is only create the controller. Do you agree?

Reply:
Sure, I have no opinion on how you split the tasks.

Reply:
Yes @andresmoschini , I had seen what you told me but whoever makes the first endpoint will be in charge of restricting it only for Doppler applications. Thank you for the feedback. :+1: